packages/contents/src/drivefs.ts

```diff
@@ -321,8 +345,8 @@ export class ContentsAPI {
     baseUrl: string,
     driveName: string,
     mountpoint: string,
-    FS: any,
-    ERRNO_CODES: any
+    FS: Pyodide.PyodideInterface['FS'],
```

Comment:
Is this interface really bound to Pyodide?

We're using this DriveFS in xeus-python-kernel (and we could technically use it with another language kernel!) and I wonder if this interface does not make it too specific.

Reply:
Hm, having some issues with `@types/emscripten`...

Reply:
The pyodide ones aren't very strong yet, either:

```ts
export declare type PyodideInterface = {
	FS: typeof FS;
	...
	ERRNO_CODES: typeof ERRNO_CODES;
};
...
declare let FS: any;
...
declare let ERRNO_CODES: any;
```

Reply:
hmmm... this works, but `ERRNO_CODES` is not hoisted,

```ts
// eslint-disable-next-line @typescript-eslint/triple-slash-reference
/// <reference path="../../../node_modules/@types/emscripten/index.d.ts" />
export type EmscriptenFS = typeof FS;
...
FS: EmscriptenFS
```

Reply:
Yep, guess it's true they're not providing a bunch of value.

I guess maybe the best way: we declare our own `any` in `@jupyerlite/types`, which we could fill in in the future. Then at least all of our references are consistent.

Reply:
because there were some constants, i ended up just keeping them in `contents` for now. I also did bring `@types/emscripten` in, which at least gives us partial coverage.